src/content/roadmap/index.md

```diff
@@ -100,3 +100,18 @@ Upgrades tend not to impact end-users except by providing better user-experience
 - The Splurge: other upgrades that don't fit well into the previous categories.
 
 We decided not to use this terminology because we wanted to use a simpler and more user-centric model. Although we use user-centric language, the vision remains the same as the one proposed by Vitalik.
+
+## What about sharding? {#what-about-sharding}
+
+Sharding is splitting up the Ethereum blockchain so that subsets of validators are only responsible for a fraction of the total data. This was originally intended to be the way for Ethereum to scale. However, layer 2 rollups have developed much faster than expected and have provided a lot of scaling already, and will provide much more after Proto-Danksharding is implemented. This means sharding the chain is no longer needed and has been dropped from the roadmap.
```

Comment:
Just a thought to make this even clearer, but not a big deal:
```suggestion
Sharding is splitting up the Ethereum blockchain so that subsets of validators are only responsible for a fraction of the total data. This was originally intended to be the way for Ethereum to scale. However, layer 2 rollups have developed much faster than expected and have provided a lot of scaling already, and will provide much more after Proto-Danksharding is implemented. This means "shard chains" are no longer needed and have been dropped from the roadmap.
```